### PR TITLE
Update tutorial to include pylint 1.6.0 output

### PR DESCRIPTION
**Pull request by Zeckie**
#Hacktoberfest (please [add label hacktoberfest-accepted](https://hacktoberfest.digitalocean.com/details#maintainers) to this PR)

## Steps

- [ ] Add yourself to CONTRIBUTORS if you are a new contributor.
- [ ] Add a ChangeLog entry describing what your PR does.
- [ ] If it's a new feature or an important bug fix, add a What's New entry in `doc/whatsnew/<current release.rst>`.
- [X] Write a good description on what the PR does.

## Description
Update the tutorial to show output from pylint 1.6.0, and use examples that are picked up by the same version. Removed messages that are no longer displayed (eg. bad-whitespace)

## Type of Changes

|   | Type |
| ------------- | ------------- |
|    | :bug: Bug fix  |
|    | :sparkles: New feature |
|    | :hammer: Refactoring  |
| ✓  | :scroll: Docs |

## Related Issue
#2118 (already closed)

### CODE REVIEW

File: doc/tutorial.rst
Comment (hippo91) [inline doc/tutorial.rst:121]: Why remove this?

File: doc/tutorial.rst
Comment (hippo91) [inline doc/tutorial.rst:148]: I do not see those warnings anymore. I think it may be due to your `pylint` settings.

File: doc/tutorial.rst
Comment (hippo91) [inline doc/tutorial.rst:190]: I do not understand why we should change this?
If you let the = sign without spaces around then the message will be emitted by `pylint`.
Why would you prefer illustrating the unnecessary semicolon from the missing spaces around operator?

**Review by hippo91 — CHANGES_REQUESTED** [review]
@Zeckie thanks for this PR.
I have got a couple of interrogations that should be addressed before merging.

File: doc/tutorial.rst
Comment (Zeckie) [reply doc/tutorial.rst:121]: Pylint no longer does the bad-whitespace check (since https://github.com/PyCQA/pylint/issues/3572 / related PRs). I thought it would be better to not have mistakes in the example that would not be picked up.

File: doc/tutorial.rst
Comment (Zeckie) [reply doc/tutorial.rst:190]: Pylint 1.6.0 does not check for spaces around the = sign.

File: doc/tutorial.rst
Comment (Zeckie) [reply doc/tutorial.rst:148]: Your comment seems to be on lines that i've removed (as I don't see those either).

I've copied the output from running pylint 1.6.0 with no custom settings.

File: doc/tutorial.rst
Comment (hippo91) [reply doc/tutorial.rst:121]: Ouups. My bad. You are totally right.

File: doc/tutorial.rst
Comment (hippo91) [reply doc/tutorial.rst:190]: Sorry for my inappropriate remark.

File: doc/tutorial.rst
Comment (hippo91) [reply doc/tutorial.rst:148]: Yes it concerns lines that have been removed. I don't understand why because with `pylint`'s master branch i still have the message `invalid-name` that is emitted.

Comment (Zeckie) [reply doc/tutorial.rst:148]: When I run it (rebased my branch with latest changes from master), it gives the same result, ie:
```
************* Module simplecaesar1
simplecaesar1.py:3:0: W0301: Unnecessary semicolon (unnecessary-semicolon)
simplecaesar1.py:1:0: C0114: Missing module docstring (missing-module-docstring)
simplecaesar1.py:5:0: C0103: Constant name "shift" doesn't conform to UPPER_CASE naming style (invalid-name)
simplecaesar1.py:9:0: C0103: Constant name "encoded" doesn't conform to UPPER_CASE naming style (invalid-name)
simplecaesar1.py:13:12: C0103: Constant name "encoded" doesn't conform to UPPER_CASE naming style (invalid-name)

------------------------------------------------------------------
Your code has been rated at 7.37/10 (previous run: 7.37/10, +0.00)
```

Note that messages are for variables `shift` and `encoded`, not `choice`, `word` or `letters`.

Comment (hippo91) [reply doc/tutorial.rst:148]: @zeckie ok you are right. Thanks.

**Review by hippo91 — APPROVED** [review]
@Zeckie it's all good. Thanks!

**Comment by Zeckie** [comment]
@hippo91 Are you able to add label `hacktoberfest-accepted` to this PR?
https://hacktoberfest.digitalocean.com/details#maintainers

Thanks, Zeckie

**Comment by hippo91** [comment]
@Zeckie i'v tried to add it when merging your PR but it seems i have not enough permissions to do that.
Maybe @Pierre-Sassoulas or @AWhetter will have a mean to do it.

**Comment by Pierre-Sassoulas** [comment]
Done, thank you for the PR ;)

**Comment by hippo91** [comment]
Thanks @Pierre-Sassoulas . How to you did that?

**Comment by Pierre-Sassoulas** [comment]
I created a new label here : create_label

**Comment by hippo91** [comment]
@Pierre-Sassoulas thanks! :+1:

**Pull request merged**
